### DISCUSSION TITLE
Payout not happened

### DISCUSSION THREAD

**Discussion by Better-Boy** [community/community#79320] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I recently received sponsorship. The money is yet to be sent out to the payout account. But the info icon says the following
![Screenshot 2023-12-16 at 6 08 07 PM](https://github.com/community/community/assets/19553184/76be2e86-9b86-4ac5-9b50-6e1950897d0f)

When will the money arrive in my account? I urgently need it.

**Comment by DHayes10** [comment, marked as answer]
Hi @Better-Boy, 

We appreciate you reaching out. 

For first payments, GitHub will remit to you all Sponsored Developer Payments 60 days after the first sponsorship is initiated, subject to the variables below. This probation period is applied to all Sponsored Developer Accounts and helps insure the integrity of the program.

For more information, please visit - https://docs.github.com/en/site-policy/github-terms/github-sponsors-additional-terms

Thank you, 
GitHub Sponsors
